src/index.js

```diff
@@ -31,8 +21,7 @@ class Notification {
         self.io.sockets[user.id] = socket;
         self.sendUnread(user.id);
```

Comment:
I believe we should take out this from here and let the user decide what actions should be taken when doing a `check-in`